ci-automation/test.sh

```diff
@@ -134,6 +136,12 @@ function test_run() {
     # Make the torcx artifacts available to test implementation
     __prepare_torcx "${arch}" "${vernum}" "${work_dir}"
 
+    # Pass PARALLEL_TESTS to the container
+    if [ -n "${PARALLEL_TESTS}" ] ; then
```

Comment:
```suggestion
    if [ -n "${PARALLEL_TESTS-}" ] ; then
```
To gracefully handle if it isn't set